### DISCUSSION TITLE
Make shortcut naming for executable file

### DISCUSSION THREAD

**Discussion by moreal** [planetarium/NineChronicles.Standalone#1641] [Ideas]
In case of go-ethereum's, the node implementation's filename is `geth`. And users can use it by running `geth`. But in Nine Chronicles, users should run `NineChronicles.Headless.Executable[.exe]` to run 9c node. I'd like to discuss the shortcut naming of the file.

**Comment by BasixKOR** [comment]
Perhaps 9cd (short for 9c-deamon)?

**Comment by U-lis** [comment]
Starting with number looks not familiar to me :sweat_smile: 
`NineChronicles.Headless.Executable` basically acts as a game server.
So I suggest `ncs` (Nine Chronicles Server) and this is much easier to type :keyboard:

**Reply by moreal** [reply]
Then I want to suggest also `ncn` (***N***ine ***C***hronicles ***N***ode) or `ncd` (***N***ine ***C***hronicles ***D***aemon) in same context 🙇🏻‍♂️
